src/include/sof/lib/perf_cnt.h

```diff
@@ -82,6 +82,37 @@ struct perf_cnt_data {
 		}							  \
 	} while (0)
 
+/**
+ * For simple performance measurement and optimization in development stage,
+ * tic-toc api is provided. Performance data are traced at each tok call,
+ * to allow fast clocks usage deviation estimation. Example:
+ *
+ * void foo(struct comp_dev *dev) {
```

Comment:
Use `\code` for code blocks inside dox. It gives much better formatting and hyperlinks to found symbols.

See https://www.doxygen.nl/manual/commands.html#cmdcode

Reply:
updated